### DISCUSSION TITLE
sw4

### DISCUSSION THREAD

**Discussion by Navids94** [geodynamics/sw4#151] [General]
Hi,
Does SW4 support the inversion of shear wave velocity? Since there is a drop in Vs at some depth in the crustal model.
In addition, I would appreciate if you could let me know if I should include the dimension and material properties of the basin in the same input file or if I should use the block command for basin modeling.
Thanks

**Comment by andersp** [comment]
The sw4mopt code can be used to optimize a material model by minimizing the misfit between filtered ground motion data and synthetics, using time-windowed full wave-form inversion. Be aware that this is a hard optimization problem and we are continuously improving the sw4mopt code.
As for basin modeling, there are many ways of specifying the material properties. The block command is the most simplistic of them. See the user's guide for more information.

**Comment by Navids94** [comment]
Thank you for your reply. The problem I am having is that I would like to use the Pfile, but I don't know how to get the material properties for all grid points?!
According to the user's guide, Sfile is capable of calculating seismic inversions. Is it necessary to use sfile for seismic inversion or can I also use pfile?
Thanks
